cs-amazon/src/main/java/io/cloudslang/content/amazon/utils/InputsUtil.java

```diff
@@ -496,6 +499,13 @@ private static void setSpecificQueryParamValue(Map<String, String> queryParamsMa
                 currentArray[index], currentArray.length > START_INDEX);
     }
 
+    private static void validateNetmask(String input) {
+        int netmaskIndex = indexOf(input, SCOPE_SEPARATOR) + ONE;
```

Comment:
when did "1" became "ONE" ? :)) I don't think the value will ever change ( so that ONE will become another number different than 1) ... either way, it's was not added in this PR so leave it as is